### ISSUE TITLE
Mystery Text discussion of web.txt

### ISSUE THREAD

**Issue by ebeshero** [newtfire/introDH-Hub#64]
Post your screenshots and discuss your findings about web.txt here!

**Comment by Stach13** [comment]
In terms of what the .txt could be written as, I think one of the most apparent aspects of it is that it's a novel written in first person. Documenting each thing the main character comes across and each dialogue encountered with other characters. Every sentence and paragraph was very descriptive to the main character's perspective, even writing some of their thoughts and motives. For example, the first image below shows a repetition of things she "does and doesn't think, want, do, etc."
![Screenshot (38)](https://user-images.githubusercontent.com/112652730/196475263-35e3f075-ce17-414b-855a-6338ec5be3ed.png)
I also found some other vocabulary to be used in the most bizarre and strangest way imaginable to describe things that happened or were happening in the moment. You'll see below what I'm talking about. I've had to read some very strange texts in the past, and this can be added to the list. 
![Screenshot (39)](https://user-images.githubusercontent.com/112652730/196475267-897080d1-df0a-4455-aa2d-2b34c743ac6b.png)

**Comment by LPHerschell** [comment]
The .txt file seems to be a first person recount/monologue of events that have taken place around the main character. Years are written to mark the time period. The format of the text seems to imply that the main character is trying to chronicle the events and interactions before them. Characters are given much detail as to what they are doing within the text. However, I noticed that while I found the word "silent" was fairly common among the text's phrases, as seen in the pictures below:
![webtxt(1)](https://user-images.githubusercontent.com/114019791/196491387-bedf57f3-9805-45f1-8e82-f2aef06ac153.png)

![webtxt(2)](https://user-images.githubusercontent.com/114019791/196491407-94bb13e1-cf38-4e9c-907e-1f2d09bd405a.png)
